app/src/main/java/org/simlar/utils/Util.java

```diff
@@ -139,4 +139,14 @@ private static SimpleDateFormat createSimpleDateFormat(final long milliSeconds)
 		}
 		return service;
 	}
+
+	public static boolean defaultTo(final Boolean value, final boolean defaultValue)
+	{
+		return value == null ? defaultValue : value;
+	}
+
+	public static int defaultTo(final Integer value, final int defaultValue)
```

Comment:
This is a special case of a generic method like [ObjectUtils.defaultIfNull](http://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/ObjectUtils.html#defaultIfNull-T-T-). Well, besides the explicit unboxing.